### PR TITLE
Auth fix

### PR DESCRIPTION
**Pull request by sdstolworthy**

### CODE REVIEW

File: src/routes/router.tsx [hunk 1]
```diff
@@ -47,7 +47,7 @@ export function LodestarRouter() {
               <Switch>
                 {/* all other routes should be considered private */}
                 <PrivateRoute path="/">
-                  <Feature name={'writer'} inactiveComponent={UnauthorizedPage}>
```
Comment (lallmon) [inline src/routes/router.tsx:50]: Where are these toggles actually managed/flipped?

Comment (sdstolworthy) [reply src/routes/router.tsx:50]: https://github.com/rht-labs/lodestar-frontend/blob/009f7f61e49d476759a5abade9b5b8af8c034a58/src/app.tsx#L100

**Comment by lallmon** [comment]
This already got merged in a different way, correct?

**Comment by oybed** [comment]
@sdstolworthy is this PR still valid? ... @philipdouble perhaps this takes care of the issues seen in bug 897

**Comment by oybed** [comment]
> @sdstolworthy is this PR still valid? ... @philipdouble perhaps this takes care of the issues seen in bug 897

NVM - looks like the one-liner that potentially fixes the auth issue have already been merged (so has the rest of this PR it appears), so most likely a no-change for this one

**Pull request closed without merging**